addons/website/views/snippets/s_cta_card.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_cta_card" name="Card Call to Action">
+    <section class="s_cta_card o_cc o_cc2 pt64 pb64">
+        <div class="container">
+            <div class="row align-items-center">
+                <div class="col-lg-9">
+                    <h2>50,000+ companies run Odoo to grow their businesses.</h2>
+                    <p class="lead">Join us and make your company a better place.</p>
+                </div>
+                <div class="col-lg-3">
+                    <div class="s_card card o_cc o_cc1 ms-auto shadow" data-name="Card" style="border-width: 0;" data-vcss="001">
+                        <div class="card-body p-4">
+                            <h5 class="card-title">What you will get</h5>
+                            <ul class="list-group list-group-flush pt-3 pb-4">
```

Comment:
I don't see the advantage of using a list group here, also colors are stuck to BS default 🥲 (see pic)
I propose to use simple lines of text, to be eventually replaced when `s_list_group`will be implemented (the snippet +  all its options and color adaptations that are needed to make the component work properly here) 😉
 
<img width="632" alt="image" src="https://github.com/user-attachments/assets/7068c35a-18e3-4a4d-86c6-5fbba4a83e2f">